Solutions/SOC-Process-Framework/Watchlists/SOC-Recommended-Actions/SocRA.json

```diff
@@ -0,0 +1,34 @@
latest patch.,15. Deploy the latest security updates for Windows, web browsers, and other applications.,16. Escalate - Contact your incident response team, or contact Microsoft support for investigation and remediation services.,17. Aditional Actions to Consider - Install and run Microsoft’s Malicious Software Removal Tool (see https://www.microsoft.com/download/malicious-software-removal-tool-details.aspx),19. Aditional Actions to Consider - Run Process Explorer and try to identify unknown running processes (see https://technet.microsoft.com/sysinternals/bb896653.aspx).,03/10/2021\r\nSharePoint / OneDrive Files shared to an External Resource - GBB Analytic,1. Identify who shared the artifact outside of the company.,2. Review what was shared. Where is the Artifact located? Was it Confidential? Was it protected with an information protection policy?,3. Identifiy who outside the company recieved the artifact.,4. Apply an Information Protection policy to the artifact to protect the artifact onward.,5. Apply (larger inititive) Information Protection policys company wide and work with business owners to apply the right level of artifact protection.,6. Recall the sharing of the artifact if possible.,8/2/2021Post-delivery detection of suspicious attachment,1. Validate the alert.,2. Inspect the attachment. Review the process that opened it and its behaviors.,3. Check for other suspicious activities in the machine timeline.,4. Locate unfamiliar processes in the process tree. Check files for prevalence, their locations, and digital signatures.,5. Submit relevant files for deep analysis and review file behaviors.,6. Identify unusual system activity with system owners.,7. Scope the incident. Find related machines, network addresses, and files in the incident graph.,8. Contain and mitigate the breach. Stop suspicious processes, isolate affected machines, decommission compromised accounts or reset passwords, block IP addresses and URLs, and install security updates.,9. Escalate - Contact your incident response team, or contact Microsoft support for investigation and remediation services.,3/18/2021Suspicious Attachment Opened,1. Validate the alert.,2. Inspect the attachment. Review the process that opened it and its behaviors.,3. Check for other suspicious activities in the machine timeline.,4. Locate unfamiliar processes in the process tree. Check files for prevalence, their locations, and digital signatures.,5. Submit relevant files for deep analysis and review file behaviors.,6. Identify unusual system activity with system owners.,7. Scope the incident. Find related machines, network addresses, and files in the incident graph.,8. Contain and mitigate the breach. Stop suspicious processes, isolate affected machines, decommission compromised accounts or reset passwords, block IP addresses and URLs, and install security updates.,9. Escalate - Contact your incident response team, or contact Microsoft support for investigation and remediation services.,2/24/2021"
```

Comment:
@rinure-msft  rawContent not be in correct format at time deploying we are facing issue. Can you please check and update it. For your reference I'm attaching screenshot. thanks!!!

![image](https://user-images.githubusercontent.com/90677907/162420501-924cdeaf-1792-4680-a1f0-61fa61049d2f.png)
![image](https://user-images.githubusercontent.com/90677907/162420501-924cdeaf-1792-4680-a1f0-61fa61049d2f.png)

Reply:
I am having trouble validating the format. I will resubmit today so you can test.